src/main/webapp/directory-page/directory.html

```diff
@@ -13,37 +13,40 @@
 
       <h1 id="header"></h1>
 
-      <form action="" onsubmit="return eventTypeChecked();">
+      <form id="filter" action="directory.html" method="GET" onsubmit="filterOnSubmit();">
```

Comment:
Get rid of the `onsubmit="filterOnSubmit()"`. This function doesn't exist in the javascript and it actually throws an error right before the page redirects.